CONTRIBUTING.md

```diff
@@ -11,7 +11,36 @@ No software is bug free. So, if you got an issue, follow these steps:
   * Clear title (make is shorter if possible).
   * Describe the issue in clear language.
   * Share error logs, screenshots and etc.
-  * To speed up the issue fixing process, send us a sample repo with the issue you faced.
+  * To speed up the issue fixing process, send us a sample repo with the issue you faced:
+  
+### Reproductions
+
+The best way to help figure out an issue you are having is to produce a minimal reproduction. A good way to do that is using Create React App:
+
+```bash
+# if necessary
+npm install -g create-react-app getstorybook
+
+create-react-app reproduction
+cd reproduction
+getstorybook
+
+# make changes to try and reproduce the problem, such as adding components + stories
+yarn start storybook
```

Comment:
I know yarn is already quite well known (because it's great) amongst the experienced FE crowd. I feel a little conflicted about recommending it. It's really important to help the novice users. Something that does not help is introducing new tools, when it's not strictly needed.

When https://github.com/facebookincubator/create-react-app uses `yarn` in their documentation you can start doing it as well.